data/transactions/logic/evalAppTxn_test.go

```diff
@@ -456,25 +456,25 @@ func TestNumInnerShallow(t *testing.T) {
 	ep.Reset()
 	ledger.NewApp(tx.Receiver, 888, basics.AppParams{})
 	ledger.NewAccount(appAddr(888), 1000000)
-	TestApp(t, pay+";int 1", ep)
-	TestApp(t, pay+pay+";int 1", ep)
-	TestApp(t, pay+pay+pay+";int 1", ep)
-	TestApp(t, pay+pay+pay+pay+";int 1", ep)
+	TestApp(t, pay+"; int 1", ep)
+	TestApp(t, pay+pay+"; int 1", ep)
+	TestApp(t, pay+pay+pay+"; int 1", ep)
+	TestApp(t, pay+pay+pay+pay+"; int 1", ep)
 	// In the sample proto, MaxInnerTransactions = 4
-	TestApp(t, pay+pay+pay+pay+pay+";int 1", ep, "too many inner transactions")
+	TestApp(t, pay+pay+pay+pay+pay+"; int 1", ep, "too many inner transactions")
 
 	ep, tx, ledger = MakeSampleEnv()
 	ledger.NewApp(tx.Receiver, 888, basics.AppParams{})
 	ledger.NewAccount(appAddr(888), 1000000)
-	TestApp(t, pay+";int 1", ep)
-	TestApp(t, pay+pay+";int 1", ep)
-	TestApp(t, pay+pay+pay+";int 1", ep)
-	TestApp(t, pay+pay+pay+pay+";int 1", ep)
+	TestApp(t, pay+"; int 1", ep)
+	TestApp(t, pay+pay+"; int 1", ep)
+	TestApp(t, pay+pay+pay+"; int 1", ep)
+	TestApp(t, pay+pay+pay+pay+"; int 1", ep)
 	// In the sample proto, MaxInnerTransactions = 4, but when pooling you get
 	// MaxTxGroupSize (here, 8) * that.
-	TestApp(t, pay+pay+pay+pay+pay+";int 1", ep)
-	TestApp(t, strings.Repeat(pay, 32)+";int 1", ep)
-	TestApp(t, strings.Repeat(pay, 33)+";int 1", ep, "too many inner transactions")
+	TestApp(t, pay+pay+pay+pay+pay+"; int 1", ep)
+	TestApp(t, strings.Repeat(pay, 32)+"; int 1", ep)
+	TestApp(t, strings.Repeat(pay, 33)+"; int 1", ep, "too many inner transactions")
```

Comment:
Maybe we should eliminate these changes, since the semi colon handling should take care of it now. Might as well have some tests that depend on that.

Reply:
Yaaaaaa, I was kind of thinking the same thing, but it'll be a bit of a pain to go back through all of the files. I'll take a look and see how many can be done with some git magic vs manual changes.

Reply:
It's not too big of a deal, as long as we have a few examples without space.

Reply:
Turns out they were pretty much all in one file so went ahead and reverted them